app/src/main/java/io/awaken/data/database/ConnectionDatabaseHelper.java

```diff
@@ -77,7 +77,6 @@ public boolean deleteConnection(int id) {
         SQLiteDatabase database = getWritableDatabase();
         String idFilter = ID_COL + "=" + "'" + id + "'";
         database.delete(CONNECTIONS_TABLE, idFilter, null);
-        database.close();
```

Comment:
Any reason why this was removed?

Reply:
We don't do it in any of the other functions so I figured it probably didn't matter.

Reply:
I think we should keep it and add it to the methods that don't have it.